mock-return-values.test.ts

```diff
@@ -0,0 +1,8 @@
+const myMock = jest.fn();
+console.log(myMock());
+// > undefined
+
+myMock.mockReturnValueOnce(10).mockReturnValueOnce('x').mockReturnValue(true);
```

Comment:
`mockReturnValueOnce()` = 
Accepts a value that will be returned for one call to the mock function. Can be **chained so that successive calls** to the mock function return different values. When there are no more mockReturnValueOnce values to use, calls will return a value specified by mockReturnValue.